[2024-10-13T19:01:18.071+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_Demo.SFTP_to_S3 manual__2024-10-13T19:01:13.576487+00:00 [queued]>
[2024-10-13T19:01:18.081+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_Demo.SFTP_to_S3 manual__2024-10-13T19:01:13.576487+00:00 [queued]>
[2024-10-13T19:01:18.082+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-10-13T19:01:18.082+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2024-10-13T19:01:18.082+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-10-13T19:01:18.093+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): SFTP_to_S3> on 2024-10-13 19:01:13.576487+00:00
[2024-10-13T19:01:18.120+0000] {standard_task_runner.py:55} INFO - Started process 222 to run task
[2024-10-13T19:01:18.125+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pipeline_Demo', 'SFTP_to_S3', 'manual__2024-10-13T19:01:13.576487+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_Demo.py', '--cfg-path', '/tmp/tmpl3s0ghcj']
[2024-10-13T19:01:18.127+0000] {standard_task_runner.py:83} INFO - Job 2: Subtask SFTP_to_S3
[2024-10-13T19:01:18.168+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/***/settings.py:250 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-13T19:01:18.271+0000] {task_command.py:389} INFO - Running <TaskInstance: pipeline_Demo.SFTP_to_S3 manual__2024-10-13T19:01:13.576487+00:00 [running]> on host c3a571b96d5f
[2024-10-13T19:01:18.428+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pipeline_Demo
AIRFLOW_CTX_TASK_ID=SFTP_to_S3
AIRFLOW_CTX_EXECUTION_DATE=2024-10-13T19:01:13.576487+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-10-13T19:01:13.576487+00:00
[2024-10-13T19:01:18.436+0000] {sftp_s3.py:187} INFO - Entered sftp_get_file function .....
[2024-10-13T19:01:18.455+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/pysftp/__init__.py:61 UserWarning: Failed to load HostKeys from /root/.ssh/known_hosts.  You will need to explicitly load HostKeys (cnopts.hostkeys.load(filename)) or disableHostKey checking (cnopts.hostkeys = None).
[2024-10-13T19:01:18.456+0000] {sftp_s3.py:210} INFO - Entered func
[2024-10-13T19:01:18.456+0000] {sftp_s3.py:73} INFO - Executing File pattern check......
[2024-10-13T19:01:18.902+0000] {transport.py:1944} INFO - Connected (version 2.0, client OpenSSH_8.4p1)
[2024-10-13T19:01:19.116+0000] {transport.py:1944} INFO - Authentication (password) successful!
[2024-10-13T19:01:19.119+0000] {logging_mixin.py:137} INFO - Connection successful to SFTP .........
[2024-10-13T19:01:19.630+0000] {sftp.py:169} INFO - [chan 0] Opened sftp connection (server version 3)
[2024-10-13T19:01:19.650+0000] {logging_mixin.py:137} INFO - objs: ['sftp_20241010.txt', 'sftp_20241010123456 .txt', 'sftp_20241011.txt', 'sftp_20241011053445.csv', 'sftp_20241011053445.txt', 'sftp_20241012.txt', 'sftp_20241012123030.txt']
[2024-10-13T19:01:19.650+0000] {sftp.py:169} INFO - [chan 0] sftp session closed.
[2024-10-13T19:01:19.652+0000] {logging_mixin.py:137} INFO - match_list: ['sftp_20241011053445.txt', 'sftp_20241012123030.txt']
[2024-10-13T19:01:19.652+0000] {logging_mixin.py:137} INFO - latest file: sftp_20241012123030.txt
[2024-10-13T19:01:19.653+0000] {sftp_s3.py:121} INFO - Latest file for pattern - sftp_\d{14}\.txt at /sftp_restricted: sftp_20241012123030.txt
[2024-10-13T19:01:19.728+0000] {transport.py:1944} INFO - Connected (version 2.0, client OpenSSH_8.4p1)
[2024-10-13T19:01:19.926+0000] {transport.py:1944} INFO - Authentication (password) successful!
[2024-10-13T19:01:19.928+0000] {sftp_s3.py:223} INFO - Connection successful to SFTP
[2024-10-13T19:01:20.333+0000] {sftp.py:169} INFO - [chan 0] Opened sftp connection (server version 3)
[2024-10-13T19:01:20.494+0000] {sftp_s3.py:227} INFO - File sftp_20241012123030.txt downloaded to local path /opt/***/tempdata/sftp_20241012123030.txt
[2024-10-13T19:01:20.513+0000] {sftp_s3.py:53} INFO - Updating run log for read operation....
[2024-10-13T19:01:20.521+0000] {sftp_s3.py:61} INFO - Error: relation "pipeline_run_log" does not exist
LINE 1: INSERT INTO pipeline_run_log VALUES( 'pipeline_Demo','manual...
                    ^

[2024-10-13T19:01:20.523+0000] {sftp.py:169} INFO - [chan 0] sftp session closed.
[2024-10-13T19:01:20.527+0000] {sftp_s3.py:18} INFO - Creating Spark session ......
[2024-10-13T19:01:51.373+0000] {sftp_s3.py:172} INFO - Moved and Renamed files
[2024-10-13T19:01:51.381+0000] {sftp_s3.py:53} INFO - Updating run log for write operation....
[2024-10-13T19:01:51.381+0000] {sftp_s3.py:61} INFO - Error: relation "pipeline_run_log" does not exist
LINE 1: INSERT INTO pipeline_run_log VALUES( 'pipeline_Demo','manual...
                    ^

[2024-10-13T19:01:51.381+0000] {sftp_s3.py:303} INFO - File transfer successful to sdevalla-buck with key sftp_file_receiver/sftp_s3_20241013190141.csv
[2024-10-13T19:01:51.382+0000] {sftp_s3.py:344} INFO - File transfer Successful
[2024-10-13T19:01:51.382+0000] {python.py:177} INFO - Done. Returned value was: sftp_file_receiver/sftp_s3_20241013190141.csv
[2024-10-13T19:01:51.401+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=pipeline_Demo, task_id=SFTP_to_S3, execution_date=20241013T190113, start_date=20241013T190118, end_date=20241013T190151
[2024-10-13T19:01:51.460+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2024-10-13T19:01:51.552+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
